### PR TITLE
Some more rules

### PR DESCRIPTION
**Pull request by gep13**
Based on the list of Rules that I added here:

https://github.com/chocolatey/package-validator/issues/3

@ferventcoder  What do you think?

### CODE REVIEW

File: src/chocolatey.package.validator/infrastructure.app/rules/ReleaseNotesNotEmptyGuideline.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+// limitations under the License.
+
+namespace chocolatey.package.validator.infrastructure.app.rules
+{
+    using NuGet;
+    using infrastructure.rules;
+
+    public class ReleaseNotesNotEmptyGuideline : BasePackageRule
+    {
+        public override string ValidationFailureMessage { get { return "Release Notes (releasenotes) are a short description of changes in each version of a package. Please include releasenotes in the nuspec."; } }
```
Comment (ferventcoder) [inline src/chocolatey.package.validator/infrastructure.app/rules/ReleaseNotesNotEmptyGuideline.cs]: Minor nit - the nuspec field is `releaseNotes` (note the casing difference). Also specify they can include a url to the release notes.

File: src/chocolatey.package.validator/infrastructure.app/rules/IconUrlMissingGuideline.cs [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+// limitations under the License.
+
+namespace chocolatey.package.validator.infrastructure.app.rules
+{
+    using NuGet;
+    using infrastructure.rules;
+
+    public class IconUrlMissingGuideline : BasePackageRule
+    {
+        public override string ValidationFailureMessage { get { return "The iconUrl should be added if there is one. Please correct this in the nuspec, if applicable."; } }
```
Comment (ferventcoder) [inline src/chocolatey.package.validator/infrastructure.app/rules/IconUrlMissingGuideline.cs]: We actually have more of a story regarding this. Can you add more of the text from the moderation notes?

>We really want to see the IconUrl being used, and some moderators want to see it being used properly, using the rawgit CDN. However it is a guideline, and something to note for a maintainer to fix up next time, not currently. Some software doesn't have a proper icon.

From that we should glean the Rawgit CDN specifically to a commit and point them to the wiki for more information on specifically how to do this. https://github.com/chocolatey/choco/wiki/CreatePackages#package-icon-guidelines

**Comment by ferventcoder** [comment]
This looks fantastic. Thanks!

File: src/chocolatey.package.validator/infrastructure.app/rules/ReleaseNotesNotEmptyGuideline.cs (hunk 1 above)
Comment (gep13) [inline src/chocolatey.package.validator/infrastructure.app/rules/ReleaseNotesNotEmptyGuideline.cs]: I actually noticed this just after I pushed the changes, but I ran out of time at lunch time :smile:

**Comment by gep13** [comment]
Ok, I think I have fixed up based on your comments, let me know what you think.

**Comment by ferventcoder** [comment]
Yep looks good :+1:

**Comment by ferventcoder** [comment]
Merged in

**Pull request merged**